firebase.json

```diff
@@ -44,4 +44,4 @@
       "port": 8088
     }
   }
-}
+}
```

Comment:
revert this change

Reply:
@davydocsurg please revert this change - there shouldn't be a diff on this line

Reply:
Fixed, this should be Prettier's format changes